python-sdk/src/astro/lineage/facets.py

```diff
@@ -23,6 +23,24 @@ class InputFileFacet(BaseFacet):
     description: str | None = None
 
 
+@attr.define
+class ExportFileFacet(BaseFacet):
```

Comment:
not related to this particular PR, but the number of classes for facets is increasing so just wondering if can do some better design

Reply:
We could have a facets folder and keep the relevant facets in particular files. Similar to what openlineage has it. https://github.com/OpenLineage/OpenLineage/tree/main/spec/facets